pftools/python/parflow/tools/ref/subsurface_conus_1.txt

```diff
@@ -1,4 +1,6 @@
-# Table from Maxwell and Condon (2016): Connections between groundwater flow and transpiration partitioning (DOI: 10.1126/science.aaf7891)
+# Table from Maxwell and Condon (2016):
+# Connections between groundwater flow and transpiration partitioning
+# DOI: 10.1126/science.aaf7891

```

Comment:
Going to start being a bit more picky about a few things.

The columns in this table don't line up.

Reply:
Please be picky! I have cleaned up the table.